notebooks/ldb-eda-structure.ipynb

```diff
@@ -0,0 +1,1941 @@
+{
```

Comment:
<p>I believe the comments I did before about the plots you used in each type of variable and about using markdowns to briefly explain the code are valid for the rest of it too.</p><p>Besides that, as I said in the beginning, it looks like the target variable has wrong values, and I guess it's because you read the wrong dataset, but overall, I think the plots and tables you used were very good. Good job!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>